hazelcast/src/test/java/com/hazelcast/config/InvalidConfigurationTest.java

```diff
@@ -474,30 +474,30 @@ private static String getDraftXml() {
                 + "</map>\n"
 
                 + "<cache name=\"default\">\n"
-                + "<key-type class-name=\"${cache-key-type-class-name}\"/>\n"
-                + "<value-type class-name=\"${cache-value-type-class-name}\"/>\n"
+                + "<key-type class-name=\"${cache-key-type-class-name}\">\n"
```

Comment:
Are these intentional? Same in other tests.

Reply:
Definitely not intentional. Should be fixed in the latest commit.